### PR TITLE
Fixed script

### LINKED ISSUE #3: docker-machine-unison require a password for ssh
I followed Readme of this repo. When I run provision.sh, it use for ssh-connection my host-machine user (not docker) and ask me a password:
```
$ /c/docker-machine-unison/provision.sh
(default) Checking whether /c/Users (vboxsf) is mounted ...
(default) Creating /c/Users ...
(default) Checking whether unison is installed ...
(default) Installing unison ...
Setting up unison ...
Done!
sergey@192.168.99.100's password:
```

Thanks in advance!

### PR DESCRIPTION
**Pull request by luckydonald**
Docker commands changed a little since back-at-the-days.

Should fix #3, too.

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -101,14 +105,17 @@ sshargs = -F $ssh_config_file
...
 
 loadConfig
 selectDockerMachine
+
+
 removeVirtualboxSharedFolders $DOCKER_MACHINE
 createDirectory $DOCKER_MACHINE $GUEST_FOLDER
 checkAndInstallUnison $DOCKER_MACHINE
 setUpUnison
-startUnison
```
Comment (fmasa) [inline provision.sh:114]: Please remove `startUnison` definition as well.

**Review by fmasa — CHANGES_REQUESTED** [review]

File: provision.sh (hunk 1 above)
Comment (fmasa) [reply provision.sh:114]: ping @luckydonald

**Comment by fmasa** [comment]
ping @luckydonald

**Comment by luckydonald** [comment]
Why is that needed?

**Comment by fmasa** [comment]
You replaced `startUnison` function call with `"$INSTALLATION_FOLDER/bin/unison" $UNISON_PROFILE_NAME`, so `startUnison` function is no longer used. So it's useless to leave its declaration there

**Comment by luckydonald** [comment]
@fmasa Edited with github's online editor, not sure if that messes something up though.

**Comment by fmasa** [comment]
Thanks :+1:

**Pull request merged**